apollo-router-core/src/lib.rs

```diff
@@ -26,31 +26,37 @@ mod cache;
 mod error;
 mod json_ext;
 mod naive_introspection;
-mod query;
 mod query_cache;
 mod query_planner;
 mod request;
 mod response;
-mod schema;
+mod spec;
 mod traits;
 
 pub use cache::*;
 pub use error::*;
 pub use json_ext::*;
 pub use naive_introspection::*;
-pub use query::*;
 pub use query_cache::*;
 pub use query_planner::*;
 pub use request::*;
 pub use response::*;
-pub use schema::*;
+pub use spec::*;
```

Comment:
side point, non blocking: considering future users of the router that will build from the libraries, does it make sense to reexport the content of every module and put everything at the root? Or should we start organizing things per module publicly (with module level documentation, etc)?

Reply:
> does it make sense to reexport the content of every module and put everything at the root?

I think this decision was made before you arrived. Some people like having a hierarchy of modules. I personally like to flatten everything at the root.

The decision is not carved on stone, this can be the subject of an ADR where the team could argue about that.